### PR TITLE
fix: prefix validation

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/555

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Review by akocbek — DISMISSED** [review]

File: patterns/roks/variables.tf [hunk 1]
```diff
@@ -13,8 +13,8 @@ variable "prefix" {
   type        = string
 
   validation {
-    error_message = "Prefix must begin with a letter and contain only lowercase letters, numbers, and - characters. Prefixes must end with a lowercase letter or number and be 13 or fewer characters."
-    condition     = can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.prefix)) && length(var.prefix) <= 13
+    error_message = "Prefix must begin with a letter and contain only lowercase letters, numbers, and - characters. Prefixes must end with a lowercase letter or number and be 16 or fewer characters."
+    condition     = can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.prefix)) && length(var.prefix) <= 16
```
[the remark this answers is not in the document]
Comment (ocofaigh) [reply patterns/roks/variables.tf:17]: If it fails, then you need to update the description to `Prefixes must be 13 or fewer characters.`

[the remark this answers is not in the document]
Comment (Aayush-Abhyarthi) [reply patterns/roks/variables.tf:17]: yes it is failing

**Review by ocofaigh — APPROVED** [review]

**Review by ocofaigh — APPROVED** [review]

**Review by akocbek — APPROVED** [review]

**Pull request merged**